### ISSUE TITLE
Command-line arguments for a prompt-toolkit based application?

### ISSUE THREAD

**Issue by simonvanderveldt** [prompt-toolkit/python-prompt-toolkit#982]
I couldn't really find anything about this, so it probably doesn't exist, but is there some built-in support/functionality for command-line arguments for a prompt-toolkit based application or should one just use one of the existing libraries that focus on building a CLI for that?

**Comment by jonathanslenders** [comment]
No, it's not prompt_toolkit's responsibility. Check out argparse, click or docopt.
